modules/howtos/pages/provisioning-cluster-resources.adoc

```diff
@@ -2,177 +2,251 @@
 :description: Provisioning cluster resources is managed at the collection or bucket level, depending upon the service affected.
 :navtitle: Provisioning Cluster Resources
 :page-aliases: ROOT:managing-clusters
+:page-toclevels: 2
+
+// API refs
+:bucket-api-reference: pass:q[BucketManager -- https://docs.couchbase.com/sdk-api/couchbase-php-client/classes/Couchbase-Cluster.html#method_buckets[`Cluster::buckets()`]]
```

Comment:
Are these something close to overoptimising for d.r.y.?  ;)

Reply:
We use these in sdk-common as well, so wanted to avoid repeating myself there 😄 .